diff -up JKQtPlotter-4.0.3/lib/jkqtplotter/graphs/jkqtprange.h.omv~ JKQtPlotter-4.0.3/lib/jkqtplotter/graphs/jkqtprange.h
diff -up JKQtPlotter-4.0.3/lib/jkqtplotter/graphs/jkqtpspecialline.h.omv~ JKQtPlotter-4.0.3/lib/jkqtplotter/graphs/jkqtpspecialline.h
diff -up JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpbaseelements.h.omv~ JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpbaseelements.h
diff -up JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpbaseplotter.h.omv~ JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpbaseplotter.h
diff -up JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpbaseplotterstyle.h.omv~ JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpbaseplotterstyle.h
--- JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpbaseplotterstyle.h.omv~	2025-10-28 02:05:08.669754614 +0100
+++ JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpbaseplotterstyle.h	2025-10-28 02:05:29.132234619 +0100
@@ -44,7 +44,6 @@
  *  \see JKQTBasePlotter, \ref jkqtpplotter_styling
  */
 class JKQTPLOTTER_LIB_EXPORT JKQTBasePlotterStyle {
-        Q_GADGET
     public:
         JKQTBasePlotterStyle();
 
diff -up JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpcoordinateaxes.h.omv~ JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpcoordinateaxes.h
diff -up JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpcoordinateaxesstyle.h.omv~ JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpcoordinateaxesstyle.h
--- JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpcoordinateaxesstyle.h.omv~	2025-10-28 02:06:25.330390328 +0100
+++ JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpcoordinateaxesstyle.h	2025-10-28 02:06:54.228183387 +0100
@@ -36,7 +36,6 @@ class JKQTBasePlotterStyle; // forward
  *
  */
 class JKQTPLOTTER_LIB_EXPORT JKQTPGridStyle {
-    Q_GADGET
 public:
     JKQTPGridStyle(bool isMajor=true);
 
@@ -76,7 +75,6 @@ public:
  *
  */
 class JKQTPLOTTER_LIB_EXPORT JKQTPCoordinateAxisStyle {
-        Q_GADGET
     public:
         JKQTPCoordinateAxisStyle();
 
diff -up JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpdatastorage.h.omv~ JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpdatastorage.h
diff -up JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpgraphsbase.h.omv~ JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpgraphsbase.h
diff -up JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpgraphsbaseerrors.h.omv~ JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpgraphsbaseerrors.h
--- JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpgraphsbaseerrors.h.omv~	2025-10-28 02:07:17.180777256 +0100
+++ JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpgraphsbaseerrors.h	2025-10-28 02:08:09.185616791 +0100
@@ -42,7 +42,6 @@
 
  */
 class JKQTPLOTTER_LIB_EXPORT JKQTPGraphErrorStyleMixin {
-        Q_GADGET
     public:
         /** \brief class contructor */
         JKQTPGraphErrorStyleMixin();
@@ -192,7 +191,6 @@ class JKQTPLOTTER_LIB_EXPORT JKQTPGraphE
     \see JKQTPYGraphErrorData, \ref jkqtplotter_graphsgroup_classstructure_mixins
  */
 class JKQTPLOTTER_LIB_EXPORT JKQTPXGraphErrorData  {
-        Q_GADGET
     public:
         /** \brief class contructor */
         JKQTPXGraphErrorData();
@@ -242,7 +240,6 @@ class JKQTPLOTTER_LIB_EXPORT JKQTPXGraph
     \see JKQTPXGraphErrorData \ref jkqtplotter_graphsgroup_classstructure_mixins
  */
 class JKQTPLOTTER_LIB_EXPORT JKQTPYGraphErrorData  {
-        Q_GADGET
     public:
         /** \brief class contructor */
         JKQTPYGraphErrorData();
@@ -290,7 +287,6 @@ class JKQTPLOTTER_LIB_EXPORT JKQTPYGraph
     \see JKQTPXGraphErrorData, JKQTPGraphErrorStyleMixin, \ref jkqtplotter_graphsgroup_classstructure_mixins
  */
 class JKQTPLOTTER_LIB_EXPORT JKQTPXGraphErrors: public JKQTPXGraphErrorData, public JKQTPGraphErrorStyleMixin {
-        Q_GADGET
     public:
         /** \brief class contructor */
         JKQTPXGraphErrors();
@@ -312,7 +308,6 @@ class JKQTPLOTTER_LIB_EXPORT JKQTPXGraph
     \see JKQTPYGraphErrorData, JKQTPGraphErrorStyleMixin, \ref jkqtplotter_graphsgroup_classstructure_mixins
  */
 class JKQTPLOTTER_LIB_EXPORT JKQTPYGraphErrors: public JKQTPYGraphErrorData, public JKQTPGraphErrorStyleMixin {
-        Q_GADGET
     public:
         /** \brief class contructor */
         JKQTPYGraphErrors();
@@ -336,7 +331,6 @@ class JKQTPLOTTER_LIB_EXPORT JKQTPYGraph
     \see JKQTPXGraphErrorData, JKQTPYGraphErrorData, JKQTPGraphErrorStyleMixin, \ref jkqtplotter_graphsgroup_classstructure_mixins
  */
 class JKQTPLOTTER_LIB_EXPORT JKQTPXYGraphErrors: public JKQTPXGraphErrorData, public JKQTPYGraphErrorData, public JKQTPGraphErrorStyleMixin {
-        Q_GADGET
     public:
         /** \brief class contructor */
         JKQTPXYGraphErrors();
diff -up JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpgraphsbasestyle.h.omv~ JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpgraphsbasestyle.h
--- JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpgraphsbasestyle.h.omv~	2025-10-28 02:08:09.190900505 +0100
+++ JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpgraphsbasestyle.h	2025-10-28 02:08:47.387381441 +0100
@@ -45,7 +45,6 @@ class JKQTBasePlotterStyle; // forward
  *  \see JKQTBasePlotter, \ref jkqtpplotter_styling
  */
 class JKQTPLOTTER_LIB_EXPORT JKQTGraphsSpecificStyleProperties {
-        Q_GADGET
     public:
         /** \brief initializes the object for the given \a type and takes some properties from the \a parent */
         JKQTGraphsSpecificStyleProperties(JKQTPPlotStyleType type, const JKQTBasePlotterStyle& parent);
@@ -114,7 +113,6 @@ class JKQTPLOTTER_LIB_EXPORT JKQTGraphsS
  *  \see JKQTBasePlotter, \ref jkqtpplotter_styling
  */
  class JKQTPLOTTER_LIB_EXPORT JKQTGeometricSpecificStyleProperties: public JKQTGraphsSpecificStyleProperties {
-     Q_GADGET
  public:
      JKQTGeometricSpecificStyleProperties(const JKQTBasePlotterStyle& parent);
      JKQTGeometricSpecificStyleProperties(const JKQTBasePlotterStyle& parent, const JKQTGraphsSpecificStyleProperties& other);
@@ -162,7 +160,6 @@ class JKQTPLOTTER_LIB_EXPORT JKQTGraphsS
  *  \see JKQTBasePlotter, \ref jkqtpplotter_styling
  */
  class JKQTPLOTTER_LIB_EXPORT JKQTAnnotationsSpecificStyleProperties: public JKQTGeometricSpecificStyleProperties {
-     Q_GADGET
  public:
      JKQTAnnotationsSpecificStyleProperties(const JKQTBasePlotterStyle& parent);
      JKQTAnnotationsSpecificStyleProperties(const JKQTBasePlotterStyle& parent, const JKQTGraphsSpecificStyleProperties& other);
@@ -208,7 +205,6 @@ class JKQTPLOTTER_LIB_EXPORT JKQTGraphsS
  *  \see JKQTBasePlotter, \ref jkqtpplotter_styling
  */
 class JKQTPLOTTER_LIB_EXPORT JKQTGraphsBaseStyle {
-        Q_GADGET
     public:
         JKQTGraphsBaseStyle(const JKQTBasePlotterStyle& parent);
 
diff -up JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpgraphsbasestylingmixins.h.omv~ JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpgraphsbasestylingmixins.h
--- JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpgraphsbasestylingmixins.h.omv~	2025-10-28 02:08:47.393620318 +0100
+++ JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpgraphsbasestylingmixins.h	2025-10-28 02:19:58.851494585 +0100
@@ -46,7 +46,7 @@ class JKQTPlotter; // forward
     .
  */
 class JKQTPLOTTER_LIB_EXPORT JKQTPGraphLineStyleMixin {
-        Q_GADGET
+        //Q_GADGET
     public:
         /** \brief class constructor */
         JKQTPGraphLineStyleMixin();
@@ -118,10 +118,10 @@ class JKQTPLOTTER_LIB_EXPORT JKQTPGraphL
         QColor getHighlightingLineColor() const;
 
 
-        Q_PROPERTY(QColor highlightingLineColor MEMBER m_highlightingLineColor READ getHighlightingLineColor WRITE setHighlightingLineColor)
-        Q_PROPERTY(QColor lineColor MEMBER m_lineColor READ getLineColor WRITE setLineColor)
-        Q_PROPERTY(Qt::PenStyle lineStyle MEMBER m_lineStyle READ getLineStyle WRITE setLineStyle)
-        Q_PROPERTY(double lineWidth MEMBER m_lineWidth READ getLineWidth WRITE setLineWidth)
+        //Q_PROPERTY(QColor highlightingLineColor MEMBER m_highlightingLineColor READ getHighlightingLineColor WRITE setHighlightingLineColor)
+        //Q_PROPERTY(QColor lineColor MEMBER m_lineColor READ getLineColor WRITE setLineColor)
+        //Q_PROPERTY(Qt::PenStyle lineStyle MEMBER m_lineStyle READ getLineStyle WRITE setLineStyle)
+        //Q_PROPERTY(double lineWidth MEMBER m_lineWidth READ getLineWidth WRITE setLineWidth)
     private:
         /** \brief graph line pen */
         QPen m_linePen;
@@ -156,7 +156,6 @@ class JKQTPLOTTER_LIB_EXPORT JKQTPGraphL
     \see JKQTPGraphDecoratedLineStyleMixin for a Mix-In for both ends
 */
 class JKQTPLOTTER_LIB_EXPORT JKQTPGraphDecoratedHeadLineStyleMixin: public JKQTPGraphLineStyleMixin {
-    Q_GADGET
 public:
     /** \brief class constructor */
     JKQTPGraphDecoratedHeadLineStyleMixin();
@@ -183,8 +182,8 @@ public:
 
 
 
-    Q_PROPERTY(JKQTPLineDecoratorStyle headDecoratorStyle MEMBER m_headDecoratorStyle READ getHeadDecoratorStyle WRITE setHeadDecoratorStyle)
-    Q_PROPERTY(double headDecoratorSizeFactor MEMBER m_headDecoratorSizeFactor READ getHeadDecoratorSizeFactor WRITE setHeadDecoratorSizeFactor)
+    //Q_PROPERTY(JKQTPLineDecoratorStyle headDecoratorStyle MEMBER m_headDecoratorStyle READ getHeadDecoratorStyle WRITE setHeadDecoratorStyle)
+    //Q_PROPERTY(double headDecoratorSizeFactor MEMBER m_headDecoratorSizeFactor READ getHeadDecoratorSizeFactor WRITE setHeadDecoratorSizeFactor)
 private:
     /** \brief head decorator style */
     JKQTPLineDecoratorStyle m_headDecoratorStyle;
@@ -207,7 +206,7 @@ private:
     \see JKQTPGraphDecoratedHeadLineStyleMixin for a Mix-In for one end (head) only
 */
 class JKQTPLOTTER_LIB_EXPORT JKQTPGraphDecoratedLineStyleMixin: public JKQTPGraphLineStyleMixin {
-    Q_GADGET
+    //Q_GADGET
 public:
     /** \brief class constructor */
     JKQTPGraphDecoratedLineStyleMixin();
@@ -247,10 +246,10 @@ public:
 
 
 
-    Q_PROPERTY(JKQTPLineDecoratorStyle headDecoratorStyle MEMBER m_headDecoratorStyle READ getHeadDecoratorStyle WRITE setHeadDecoratorStyle)
-    Q_PROPERTY(JKQTPLineDecoratorStyle tailDecoratorStyle MEMBER m_tailDecoratorStyle READ getTailDecoratorStyle WRITE setTailDecoratorStyle)
-    Q_PROPERTY(double headDecoratorSizeFactor MEMBER m_headDecoratorSizeFactor READ getHeadDecoratorSizeFactor WRITE setHeadDecoratorSizeFactor)
-    Q_PROPERTY(double tailDecoratorSizeFactor MEMBER m_tailDecoratorSizeFactor READ getTailDecoratorSizeFactor WRITE setTailDecoratorSizeFactor)
+    //Q_PROPERTY(JKQTPLineDecoratorStyle headDecoratorStyle MEMBER m_headDecoratorStyle READ getHeadDecoratorStyle WRITE setHeadDecoratorStyle)
+    //Q_PROPERTY(JKQTPLineDecoratorStyle tailDecoratorStyle MEMBER m_tailDecoratorStyle READ getTailDecoratorStyle WRITE setTailDecoratorStyle)
+    //Q_PROPERTY(double headDecoratorSizeFactor MEMBER m_headDecoratorSizeFactor READ getHeadDecoratorSizeFactor WRITE setHeadDecoratorSizeFactor)
+    //Q_PROPERTY(double tailDecoratorSizeFactor MEMBER m_tailDecoratorSizeFactor READ getTailDecoratorSizeFactor WRITE setTailDecoratorSizeFactor)
 private:
     /** \brief head decorator style */
     JKQTPLineDecoratorStyle m_headDecoratorStyle;
@@ -278,7 +277,7 @@ private:
     .
  */
 class JKQTPLOTTER_LIB_EXPORT JKQTPGraphSymbolStyleMixin {
-        Q_GADGET
+        //Q_GADGET
     public:
         /** \brief class constructor */
         JKQTPGraphSymbolStyleMixin();
@@ -314,11 +313,11 @@ class JKQTPLOTTER_LIB_EXPORT JKQTPGraphS
 
 
 
-        Q_PROPERTY(JKQTPGraphSymbols symbolType MEMBER m_symbolType READ getSymbolType WRITE setSymbolType)
-        Q_PROPERTY(QColor symbolColor MEMBER m_symbolColor READ getSymbolColor WRITE setSymbolColor)
-        Q_PROPERTY(QColor symbolFillColor MEMBER m_symbolFillColor READ getSymbolFillColor WRITE setSymbolFillColor)
-        Q_PROPERTY(double symbolSize MEMBER m_symbolSize READ getSymbolSize WRITE setSymbolSize)
-        Q_PROPERTY(double symbolLineWidth MEMBER m_symbolLineWidth READ getSymbolLineWidth WRITE setSymbolLineWidth)
+        //Q_PROPERTY(JKQTPGraphSymbols symbolType MEMBER m_symbolType READ getSymbolType WRITE setSymbolType)
+        //Q_PROPERTY(QColor symbolColor MEMBER m_symbolColor READ getSymbolColor WRITE setSymbolColor)
+        //Q_PROPERTY(QColor symbolFillColor MEMBER m_symbolFillColor READ getSymbolFillColor WRITE setSymbolFillColor)
+        //Q_PROPERTY(double symbolSize MEMBER m_symbolSize READ getSymbolSize WRITE setSymbolSize)
+        //Q_PROPERTY(double symbolLineWidth MEMBER m_symbolLineWidth READ getSymbolLineWidth WRITE setSymbolLineWidth)
     private:
         /** \brief which symbol to use for the datapoints */
         JKQTPGraphSymbols m_symbolType;
@@ -393,7 +392,7 @@ class JKQTPLOTTER_LIB_EXPORT JKQTPGraphS
     .
  */
 class JKQTPLOTTER_LIB_EXPORT JKQTPGraphFillStyleMixin {
-        Q_GADGET
+        //Q_GADGET
     public:
         /** \brief class constructor */
         JKQTPGraphFillStyleMixin();
@@ -432,8 +431,8 @@ class JKQTPLOTTER_LIB_EXPORT JKQTPGraphF
         /** \brief sets a fill transformation */
         void setFillTransform(const QTransform& b);
 
-        Q_PROPERTY(Qt::BrushStyle fillStyle MEMBER m_fillStyle READ getFillStyle WRITE setFillStyle)
-        Q_PROPERTY(QColor fillColor MEMBER m_fillColor READ getFillColor WRITE setFillColor)
+        //Q_PROPERTY(Qt::BrushStyle fillStyle MEMBER m_fillStyle READ getFillStyle WRITE setFillStyle)
+        //Q_PROPERTY(QColor fillColor MEMBER m_fillColor READ getFillColor WRITE setFillColor)
     private:
         /** \brief fill style of the graph */
         QBrush m_fillBrush;
@@ -450,7 +449,7 @@ class JKQTPLOTTER_LIB_EXPORT JKQTPGraphF
  *  \ingroup jkqtplotter_basegraphs_stylemixins
 */
 class JKQTPLOTTER_LIB_EXPORT JKQTPGraphLineAndFillStyleMixin: public JKQTPGraphFillStyleMixin, public JKQTPGraphLineStyleMixin {
-    Q_GADGET
+    //Q_GADGET
 public:
     /** \brief class constructor */
     JKQTPGraphLineAndFillStyleMixin();
@@ -470,8 +469,8 @@ public:
     bool doFillCurve() const;
 
 
-    Q_PROPERTY(bool drawLine MEMBER m_drawLine READ getDrawLine WRITE setDrawLine)
-    Q_PROPERTY(bool fillCurve MEMBER m_fillCurve READ getFillCurve WRITE setFillCurve)
+    //Q_PROPERTY(bool drawLine MEMBER m_drawLine READ getDrawLine WRITE setDrawLine)
+    //Q_PROPERTY(bool fillCurve MEMBER m_fillCurve READ getFillCurve WRITE setFillCurve)
 private:
     /** \brief indicates whether to draw a line on the circumference of the described area (i.e. along the data points from \c xColumn and \c yColumn as well as \c xColumn and \c yColumn2 or not */
     bool m_drawLine;
@@ -491,7 +490,7 @@ private:
     .
  */
 class JKQTPLOTTER_LIB_EXPORT JKQTPGraphTextStyleMixin {
-        Q_GADGET
+        //Q_GADGET
     public:
         /** \brief class constructor */
         JKQTPGraphTextStyleMixin(JKQTBasePlotter *parent);
@@ -518,9 +517,9 @@ class JKQTPLOTTER_LIB_EXPORT JKQTPGraphT
 
 
 
-        Q_PROPERTY(QColor textColor MEMBER m_textColor READ getTextColor WRITE setTextColor)
-        Q_PROPERTY(double textFontSize MEMBER m_textFontSize READ getTextFontSize WRITE setTextFontSize)
-        Q_PROPERTY(QString textFontName MEMBER m_textFontName READ getTextFontName WRITE setTextFontName)
+        //Q_PROPERTY(QColor textColor MEMBER m_textColor READ getTextColor WRITE setTextColor)
+        //Q_PROPERTY(double textFontSize MEMBER m_textFontSize READ getTextFontSize WRITE setTextFontSize)
+        //Q_PROPERTY(QString textFontName MEMBER m_textFontName READ getTextFontName WRITE setTextFontName)
     private:
         /** \brief color of the text */
         QColor m_textColor;
diff -up JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpimagetools.h.omv~ JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpimagetools.h
--- JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpimagetools.h.omv~	2025-10-28 02:09:29.403250197 +0100
+++ JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpimagetools.h	2025-10-28 02:10:22.345568838 +0100
@@ -41,7 +41,6 @@ class JKQTBasePlotter; // forward
     \ingroup jkqtplotter_imagelots_tools
  */
 class JKQTPLOTTER_LIB_EXPORT JKQTPColorPaletteStyleAndToolsMixin {
-        Q_GADGET
     public:
 
         JKQTPColorPaletteStyleAndToolsMixin(JKQTBasePlotter *parent);
@@ -251,7 +250,6 @@ class JKQTPLOTTER_LIB_EXPORT JKQTPColorP
     \ingroup jkqtplotter_imagelots_tools
  */
 class JKQTPLOTTER_LIB_EXPORT JKQTPColorPaletteWithModifierStyleAndToolsMixin : public JKQTPColorPaletteStyleAndToolsMixin {
-        Q_GADGET
     public:
 
 
diff -up JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpinterfacecimg.h.omv~ JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpinterfacecimg.h
diff -up JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpinterfaceopencv.h.omv~ JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpinterfaceopencv.h
diff -up JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpkeystyle.h.omv~ JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpkeystyle.h
--- JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpkeystyle.h.omv~	2025-10-28 02:10:32.742290699 +0100
+++ JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtpkeystyle.h	2025-10-28 02:10:43.723566312 +0100
@@ -34,7 +34,6 @@ class JKQTBasePlotterStyle; // forward
  *  \see JKQTBasePlotter, JKQTBasePlotterStyle, \ref jkqtpplotter_styling
  */
 class JKQTPLOTTER_LIB_EXPORT JKQTPKeyStyle  {
-        Q_GADGET
     public:
         JKQTPKeyStyle();
         JKQTPKeyStyle(const JKQTBasePlotterStyle& baseStyle);
diff -up JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtplotter.h.omv~ JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtplotter.h
diff -up JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtplotter_imexport.h.omv~ JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtplotter_imexport.h
diff -up JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtplotterstyle.h.omv~ JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtplotterstyle.h
--- JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtplotterstyle.h.omv~	2025-10-28 02:11:05.430692446 +0100
+++ JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtplotterstyle.h	2025-10-28 02:11:18.629831342 +0100
@@ -37,7 +37,6 @@
  *  \see JKQTPlotter, \ref jkqtpplotter_styling
  */
 class JKQTPLOTTER_LIB_EXPORT JKQTPlotterStyle {
-    Q_GADGET
 public:
     /** \brief initialize with default values */
     JKQTPlotterStyle();
diff -up JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtptools.h.omv~ JKQtPlotter-4.0.3/lib/jkqtplotter/jkqtptools.h
